automation/test.sh

```diff
@@ -443,7 +443,7 @@ if [[ -z ${KUBEVIRT_E2E_FOCUS} && -z ${KUBEVIRT_E2E_SKIP} && -z ${label_filter}
     export KUBEVIRT_E2E_PARALLEL=false
     label_filter='((sig-compute && Serial) && !(GPU,VGPU,sig-compute-migrations) && !(software-emulation, SEV, SEVES))'
   elif [[ $TARGET =~ sig-compute-parallel ]]; then
-    label_filter='(sig-compute && !(Serial,GPU,VGPU,sig-compute-migrations) && !(software-emulation, SEV, SEVES))'
+    label_filter='(sig-compute && !(Serial,GPU,VGPU,sig-compute-migrations,sig-storage) && !(software-emulation, SEV, SEVES))'
```

Comment:
are there tests with both storage and compute?
Should we cover `sig-compute` [serial|conformance] lane too?

Reply:
Here I am filtering tests that are tagged with both labels.

Reply:
Yes, what I am suggesting is to do the same for `sig-compute` (without `parallel`) at line 449 below

Reply:
Can we do follow up? @fossedihelm